facade/initial/initialNodeFacade.go

```diff
@@ -341,6 +341,11 @@ func (inf *initialNodeFacade) GetTokenSupply(_ string) (*api.ESDTSupply, error)
 	return nil, errNodeStarting
 }
 
+// GetGenesisNodesPubKeys returns nothing
```

Comment:
actually it returns something - the node is starting error

Reply:
:+1: